plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -202,6 +205,51 @@ unspecified, the javadoc will be downloaded from the artifact location if availa
       </xs:complexType>
    </xs:element>
 
+   <xs:element name="enablement">
+      <xs:annotation>
+         <xs:documentation>
```

Comment:
It's a PDE extension, I guess.  But the above is what's done to reference the expression language in various extension points like [`org.eclipse.ui.handlers`](http://git.eclipse.org/c/platform/eclipse.platform.ui.git/tree/bundles/org.eclipse.ui/schema/handlers.exsd?h=master#n30).